### DISCUSSION TITLE
Video URL not found

### DISCUSSION THREAD

**Discussion by gukstan** [pystardust/ani-cli#562] [General] [answered]
So I'm getting this error since today. I used ani-cli for 3 or 4 days without running into any problem, but now it's just not working. Whenever I select the episode, it displays "Video URL not found". I'm on Ubuntu 20.04.3.

Here's exactly how it displays:
Getting data for episode 2 
Video URL not found

**Comment by mrfluffy-dev** [comment]
It is being worked on at issue #560

**Comment by 71zenith** [comment, marked as answer]
We are aware of the issue. Pls refer to https://github.com/pystardust/ani-cli/issues/560 for further details.
